splunk_add_on_ucc_framework/global_config_validator.py

```diff
@@ -58,7 +65,7 @@ def _validate_config_against_schema(self) -> None:
         Validates config against JSON schema.
         Raises jsonschema.ValidationError if config is not valid.
         """
-        schema_path = os.path.join(self._source_dir, "schema", "schema.json")
+        schema_path = os.path.join(self._internal_root_dir, "schema", "schema.json")
```

Comment:
Why this change?

Reply:
I have just renamed the `_source_dir` to `_internal_root_dir` in `__init__` method.